bundler.go

```diff
@@ -48,6 +48,8 @@ type Configuration struct {
 	// The path of the go binary
 	// Best is to leave it empty. Default value is "go"
 	GoBinaryPath string `json:"go_binary_path"`
+	// The path of the resouces, default is resouces
```

Comment:
Default should be `resources` not `resouces`

Reply:
done.